source/patterns/@aws-solutions-constructs/core/test/cloudfront-distribution-s3-helper.test.ts

```diff
@@ -158,85 +157,6 @@ test('test cloudfront with no security headers ', () => {
   });
 });
 
-test('test cloudfront override cloudfront custom domain names ', () => {
```

Comment:
Does this test no longer pass? Does that indicate a change in functionality of Constructs using this?

Reply:
Unit tests are tricky because it doesn't allow to turn on the feature flag in CDK v1 and CDK v2 doesn't allow to turn off the flag. The feature flag in question here is [this](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/cx-api/lib/features.ts#L161) one which produces different expected results 

from CDK v1

```
        MinimumProtocolVersion: "TLSv1.2_2019",
```

-vs- CDK v2

```
        MinimumProtocolVersion: "TLSv1.2_2021",
```

Reply:
Unit tests are tricky because it doesn't allow to turn on the feature flag in CDK v1 and CDK v2 doesn't allow to turn off the flag. The feature flag in question here is [this](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/cx-api/lib/features.ts#L161) one which produces different expected results 

from CDK v1

```
        MinimumProtocolVersion: "TLSv1.2_2019",
```

-vs- CDK v2

```
        MinimumProtocolVersion: "TLSv1.2_2021",
```

Reply:
It's using toHaveResourceLike(), why not leave the test in but reduce the information it's looking for to a minimum?  Just confirming that myDomains was there would show the custom domain is recognized and give us more value than dropping the whole test.

Reply:
Goode idea! will do that